root/static/scripts/guess-case/modes.js

```diff
@@ -282,3 +283,27 @@ exports.Sentence = assign({}, DefaultMode, {
     {url: {href: 'https://musicbrainz.org/doc/Style/Language', target: '_blank'}}
   ),
 });
+
+exports.Turkish = assign({}, DefaultMode, {
+  description: l(
+    'This mode capitalises almost all words of a sentence, the words ' +
+    'which are restricted to be capitalised are lowercased. Some words, often proper nouns, ' +
+    'may need to be manually fixed according to the {url|Turkish ' +
+    'language guidelines}.',
+    {url: {href: 'https://musicbrainz.org/doc/Style/Language/Turkish', target: '_blank'}}
+  ),
+
+  isSentenceCaps() {
+    return false;
+  },
+
+  runPostProcess(is) {
+    return DefaultMode.runPostProcess(is)
+      .replace(/I/g, 'i') //replace 'I' with 'i' as 'I' is not valid in Turkish
+      //replace of Turkish lowercase words if they have been used at the end of sentence and capitalised
+      .replace(/Mı/g, 'mı')
+      .replace(/Mi/g, 'mi')
+      .replace(/Mu/g, 'mu')
+      .replace(/Mü/g, 'mü');
```

Comment:
What is the purpose of these replacements?
Would it be fine with [Mevlevi Ayin Müzikleri](http://musicbrainz.org/recording/d051add7-76fe-42dc-981f-1cacf8570ad0)?